models/trello_schema.yml

```diff
@@ -0,0 +1,44 @@
+version: 2
+
+sources:
+  - name: trello_test
+    database: trello_test
```

Comment:
Remove hard coded database. Just delete the whole line, dbt will handle this in our environment.

Reply:
resolved.